front/lib/auth.ts

```diff
@@ -22,7 +22,7 @@ const {
   NODE_ENV,
 } = process.env;
 
-export type RoleType = "admin" | "builder" | "user" | "none";
+export type RoleType = "admin" | "builder" | "user" | "none" | "super_user";
```

Comment:
If we don’t use it yet, maybe no need to introduce it, but we def want a isDustSuperUser as we have a isAdmin

Reply:
👍